pallets/migration/src/lib.rs

```diff
@@ -351,5 +418,35 @@ pub mod pallet {
 				.into(),
 			)
 		}
+
+		/// This extrinsic disables the call-filter. After this has been called the chain will accept
+		/// all calls again.
+		#[pallet::weight(<T as pallet::Config>::WeightInfo::finalize())]
```

Comment:
For my education, when do we know that we can call this extrinsic and flag the migration as finished?

Reply:
You can query the state of the chain.